File: SecondYear/praveer7398_PraveerNandan_2226ce1011_2/Week_2/check_palindrome.java
import java.util.*;
public class check_palindrome {    
    public static void main(String[] args) {
        try (Scanner sc = new Scanner(System.in)) {
            int n=sc.nextInt();


    int rev=0;        

    int num=n;        

    while(n!=0){   

             int rem=n%10;  

             rev=rev*10+rem;            

             n=n/10;

      }       

    if(rev==num){            

               System.out.print("True");        

}        

    else{           

               System.out.print("False");      

 }
        }    

} 

}

File: SecondYear/ApoorvaSri123_ApoorvaSrivastava_2226it1116_2/Week_2/check_palindrome.c
#include <stdio.h>

int main() {
    int n;
    printf("Enter a number: ");
    scanf("%d", &n);

    int rev = 0;
    int num = n;

    while (n != 0) {
        int rem = n % 10;
        rev = rev * 10 + rem;
        n = n / 10;
    }

    if (rev == num) {
        printf("True\n");
    } else {
        printf("False\n");
    }

    return 0;
}
